### DISCUSSION TITLE
FlexSearch in 404 page

### DISCUSSION THREAD

**Discussion by cnichte** [thuliteio/doks#1198] [Q&A]
Hi, I want to display the FlexSearch input in the body of the 404 page, although it is per default accessible via the top menu bar. I have noticed that people do not want to find the above entry. I want to make it easier to use the search when content is not found. Any advice is welcome to get this to life  as always :-)

**Comment by h-enk** [comment]
If you'd like to add the search input in the body of the page, how would you want to display the results? Also, note that you would have two search functions with a different UX (if you keep the header bar search in place) — could be confusing?

The current default 404 page only states that the requested page could not be found. 

![Snag_1126ff46](https://github.com/gethyas/doks/assets/3902872/a3349b36-f690-4d4d-b401-e609a314e881)

A clear action on what to do next would surely be beneficial to the user/visitor. Think of a link or button  to the homepage and/or suggest using the search function (from the navigation bar).

To me, that would be sufficient
